### PR TITLE
Update liblouis to 3.13.0

### PR DESCRIPTION
**Pull request by LeonarddeR**
### Link to issue number:
None

### Summary of the issue:
Liblouis 3.13.0 has just been released.

### Description of how this pull request fixes the issue:
Update to liblouis 3.13.0, which apart from a new German braille table has no new or removed tables. The new tables haven't been added yet, as I"m not sure how they fit in the current list of tables. Any feedback would be appreciated.

### Testing performed:
T.b.d.

### Known issues with pull request:
None

### Change log entry:
* Changes
    + Updated liblouis braille translator to [3.13.0](https://github.com/liblouis/liblouis/releases/tag/v3.13.0)

### CODE REVIEW

**Comment by zstanecic** [comment]
Hi Leonard,

Apart from change you mentioned, there is new braille table for Russian, (improved ru.ctb. 

I recommend merging this.

*	Added Unicode characters,
*	Fixed rendering of punctuation for Russian
*	Added displaying of Hebrew, Greek and accented characters in the Russian table

I hope that helps.

**Comment by leonardder** [comment]
@zstanecic I assume that's already in and doesn't require a change?

**Comment by zstanecic** [comment]
Yes, you don’t need to change anything.

In scope of this pull request, please try to add the malay braille table, too.

As I see, there is this table in.

**Comment by zstanecic** [comment]
Oops, 

My fault. 

There is no malay braille table yet.

There are just changes to let implement it.

But, I suggest doing one thing.

In the braille tables.py

Rename 

Russian braille for computer code to

Russian computer braille

**Comment by DrSooom** [comment]
@leonardder: I suggest to add "(improved back-translation, experimental)" to the two new German entries.
CC: @Adriani90

**Comment by leonardder** [comment]
I think it is enough to reference the new version of liblouis. If people want to know what's new, they can refer to the liblouis changelog. May be it makes sense to refer to their changelog from ours.

**Comment by bdorer** [comment]
That's a great idea to reference liblouis' news file.

**Comment by DrSooom** [comment]
But then please don't forget to save these pages in the [Wayback Machine](https://archive.org/web/).

**Comment by zstanecic** [comment]
yep, good.
leonardder,
what's the progress of this?

**Comment by leonardder** [comment]
It's awaiting review from one of the core devs.

**Comment by zstanecic** [comment]
It is important for 2020.1, as it fixes the Russian table.

Cc @mdcurran

**Comment by mdcurran** [comment]
You'll be wanting to CC @michaelDCurran I believe and not myself. 🙂

**Comment by zstanecic** [comment]
Yes, thanks for correcting me

**Comment by leonardder** [comment]
@feerrenrut Could you please assess importance of this for 2020.1? I think it doesn't harm us to have it in.

**Comment by feerrenrut** [comment]
As with any dependency update, it is best if it can get some testing time on alpha to reduce the risk of delaying the release. Since I'm not familiar with the full scope of changes, it's very hard to make an exception to rush this into the release. We are planning on starting the release process for 2020.2 shortly after release of 2020.1, which means it will not be long before this is released anyway if it doesn't go into 2020.1.

**Comment by leonardder** [comment]
It looks like the NVDA 2020.2 merge window has started. Could this be merged to have broad testing?

**Comment by DrSooom** [comment]
@leonardder: See issue #10912. Unsure if this PR will solve that issue too. See also [my previous comment](https://github.com/nvaccess/nvda/pull/10832#issuecomment-593723179).

**Review by feerrenrut — APPROVED** [review]

**Comment by feerrenrut** [comment]
Thanks for the reminder @leonardder

**Pull request merged**
